tests/image-stream/tester.js

```diff
@@ -35,12 +34,16 @@ const doFilesContainTheSameData = (file1, file2) => {
 }
 
 const deleteIfExists = (file) => {
-  return Bluebird.try(function () {
-    if (fileExists(file)) {
-      return fs.unlinkAsync(file)
+  return new Bluebird((resolve, reject) => {
+    try {
+      fs.unlinkSync(file)
```

Comment:
It seems strange to use `unlinkSync` and transform it to a `Promise` instead of `unlinkAsync`.